### ISSUE TITLE
[Feature request]: 

### ISSUE THREAD

**Issue by casesolved-co-uk** [yrestom/POS-Awesome#210]
Does POS Awesome provide/work with a payment provider integration that supports in-person card payments, i.e. with a physical terminal for contactless payments?

For example Stripe [Terminal](https://stripe.com/en-gb/terminal) / Reader?

**Comment by yrestom** [comment]
Currently not supported

**Comment by casesolved-co-uk** [comment]
Hi Yousef,

Okay, I will in the next month or two be releasing a full stripe integration with ERPNext that allows [Stripe Terminals](https://stripe.com/docs/terminal/payments/setup-reader/bbpos-wisepos-e) and works with the ERPNext POS interface. If you wanted to work together, I could (in the long-term) also support your module?

It is also designed to support stripe's cheap bluetooth modules on a phone/tablet, but this requires a native mobile app which I have not started. Depending on the demand, I can also develop with a webview for your/ERPNext POS interface for a completely mobile POS system!

Pricing for my stripe module on frappe cloud will be free with a very small percentage added on each transaction.

Let me know if you would be interested!

![Screen Shot 2022-11-29 at 20 44 19](https://user-images.githubusercontent.com/110036763/213703049-ae6bd326-4846-4df3-83cb-73b2fa26a4c4.png)

**Comment by yrestom** [comment]
Your Module can support other devices in addition to Strip?

**Comment by casesolved-co-uk** [comment]
No, only Stripe, but they support many different payment types in many different countries, useful for frappe cloud. You could ask them if Turkey is planned. Otherwise you'll need to implement another gateway.

https://stripe.com/global

**Comment by casesolved-co-uk** [comment]
The main reason is, the payment gateways are not linked to payment methods in ERPNext. If we made that change, you could use any payment gateway.

My integration works through payment requests, so once the request is created, it can be picked up and processed by any integration. Would need a few hooks adding too.

**Comment by yrestom** [comment]
@casesolved-co-uk it seems interesting, please, when you have time contact me to discuss it
